Chummer/data/vehicles.xml

```diff
@@ -10203,6 +10203,236 @@
+          <control>Remote</control>
+          <flexibility>Turret</flexibility>
+          <size>Heavy</size>
+          <visibility>External</visibility>
+        </weaponmount>
+      </weaponmounts>
+      <weapons>
+        <weapon>
+          <name>Krupp Munitions 3E Firefighting Cannon</name>
+        </weapon>
+      </weapons>
+    </vehicle>
+    <vehicle>
+      <id>7d708ed4-02ce-4047-91bb-6bce6bee03aa</id>
+      <name>THW MzKW C</name>
+      <source>SOTG</source>
+      <page>16</page>
+      <accel>2</accel>
+      <armor>10</armor>
+      <avail>8</avail>
+      <body>14</body>
+      <category>Cars</category>
+      <cost>125000</cost>
+      <handling>5/5</handling>
+      <pilot>3</pilot>
+      <sensor>5</sensor>
+      <speed>4</speed>
+      <seats>5</seats>
+      <gears>
+        <gear>
+          <name>Sensor Array</name>
+          <rating>1</rating>
+          <maxrating>6</maxrating>
+        </gear>
+      </gears>
+      <mods>
+        <name rating="2">Anti-Theft System</name>
+        <name>Off-Road Suspension</name>
+        <name>Amenities (Low)</name>
+        <name rating="1">Life Support</name>
+        <name>Multifuel Engine</name>
+        <name>Retrans Unit</name>
+        <name rating="6">Special Armor Modification (Radiation)</name>
+        <name rating="6">Special Armor Modification (Chemical)</name>
+        <name>Extra Entry/Exit Points</name>
+      </mods>
+      <weaponmounts>
+        <weaponmount>
+          <control>Manual</control>
+          <flexibility>Flexible</flexibility>
+          <size>Heavy</size>
+          <visibility>External</visibility>
+        </weaponmount>
+      </weaponmounts>
+    </vehicle>
+    <vehicle>
+      <id>143ec17c-1b87-4cb8-aea7-0a8af46f0f76</id>
+      <name>ZET Commando</name>
+      <source>SOTG</source>
+      <page>16</page>
+      <accel>3</accel>
+      <armor>14</armor>
+      <avail>12R</avail>
+      <body>16</body>
+      <category>Cars</category>
+      <cost>170000</cost>
+      <handling>3/4</handling>
+      <pilot>4</pilot>
+      <sensor>4</sensor>
+      <speed>4</speed>
+      <seats>5</seats>
+      <gears>
+        <gear>
+          <name>Sensor Array</name>
+          <rating>1</rating>
+          <maxrating>6</maxrating>
+        </gear>
+      </gears>
+      <mods>
+        <name rating="2">Anti-Theft System</name>
+        <name>Off-Road Suspension</name>
+        <name>Amenities (Low)</name>
+        <name rating="1">Life Support</name>
+        <name>Multifuel Engine</name>
+        <name>Retrans Unit</name>
+        <name rating="6">Special Armor Modification (Radiation)</name>
+        <name rating="6">Special Armor Modification (Chemical)</name>
+      </mods>
+      <weaponmounts>
+        <weaponmount>
+          <control>Remote</control>
+          <flexibility>Turret</flexibility>
+          <size>Heavy</size>
+          <visibility>External</visibility>
+        </weaponmount>
+      </weaponmounts>
+    </vehicle>
```

Comment:
This vehicle comes with a remote, external, turret heavy weapon mount, but no weapon that's part of it. Is that intentional?

Reply:
It is intentional, since that is, what is written in the Datapuls SOTA2080